test/reporters/multiple.js

```diff
@@ -0,0 +1,98 @@
+var fs = require('fs');
+var os = require('os');
+var path = require('path');
+var Mocha = require('../../');
+var Suite = Mocha.Suite;
+var Runner = Mocha.Runner;
+var Test = Mocha.Test;
+var should = require('should');
+
+describe('multiple reporters', function() {
+  var suite;
+  var runner;
+  var specReporter;
+  var jsonReporter;
```

Comment:
Declaring these vars in this scope has apparently no purpose – probably leftover code.